src/System.Net.Sockets/src/System/Net/Sockets/SocketPal.Linux.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics;
+
+namespace System.Net.Sockets
+{
+    internal static partial class SocketPal
+    {
+        public const bool SupportsMultipleConnectAttempts = true;
+
+        public static unsafe void PrimeForNextConnectAttempt(int fileDescriptor, int socketAddressLen)
```

Comment:
This is only ever used by elsewhere in SocketPal, right?  If so, it could be private.